### PR TITLE
Lower Norfair West Crystal Flash strats

### PR DESCRIPTION
**Pull request by blkerby**

### CODE REVIEW

File: region/lowernorfair/west/Golden Torizo's Room.json [hunk 1]
```diff
@@ -607,6 +616,11 @@
         {"heatFrames": 200}
       ]
     },
+    {
+      "link": [2, 2],
+      "name": "Crystal Flash",
+      "requires": ["h_canHeatedCrystalFlash"]
```
Comment (osse101) [inline region/lowernorfair/west/Golden Torizo's Room.json:622]: It doesn't look like you can avoid a GT hit if he's alive.

File: region/lowernorfair/west/Screw Attack Room.json [hunk 2]
```diff
@@ -203,6 +207,16 @@
...
   "strats": [
+    {
+      "link": [1, 1],
+      "name": "Crystal Flash",
+      "requires": [
+        {"obstaclesNotCleared": ["C"]},
+        {"heatFrames": 10},
+        "h_canHeatedCrystalFlash",
+        {"heatFrames": 10}
+      ]
```
Comment (osse101) [inline region/lowernorfair/west/Screw Attack Room.json:218]:
```suggestion
      ],
      "clearsObstacles": ["B"]
```

File: region/lowernorfair/west/Screw Attack Room.json [hunk 3]
```diff
@@ -736,6 +770,14 @@
       "note": "Carry a shinecharge down though the bomb blocks and shinespark back up.",
       "devNote": "There is no actual runway. Obstacle C is indicating that Samus has a shinecharge ready to be used."
     },
+    {
+      "link": [4, 4],
+      "name": "Crystal Flash",
+      "requires": [
+        {"obstaclesNotCleared": ["C"]},
+        "h_canHeatedCrystalFlash"
+      ]
```
Comment (osse101) [inline region/lowernorfair/west/Screw Attack Room.json:779]:
```suggestion
      ],
      "clearsObstacles": ["B"]
```

File: region/lowernorfair/west/Golden Torizo's Room.json (hunk 1 above)
Comment (blkerby) [reply region/lowernorfair/west/Golden Torizo's Room.json]: It seems to work if you CF immediately on entry and then morph and roll under. Added a note

**Pull request merged**